Source/Core/PolylineVolumeGeometry.js

```diff
@@ -163,7 +163,13 @@ define([
         }
 
         if (vertexFormat.tangent || vertexFormat.binormal) {
-            geometry = GeometryPipeline.computeBinormalAndTangent(geometry);
+            try {
+                geometry = GeometryPipeline.computeBinormalAndTangent(geometry);
+            } catch (e) {
+                console.log('Unable to compute tangents and binormals for polyline volume geometry');
```

Comment:
We could use `oneTimeWarning` and log out the url to the GitHub issue, so if people run into it (doubtful) they can comment or +1 it.